eclass/tree-sitter-grammar.eclass

```diff
@@ -35,6 +36,43 @@ RESTRICT+=" !test? ( test )"
 # Used to override upstream tag name if tagged differently, e.g. most releases
 # are v${PV} but some are tagged as rust-${PV}.
 
+# @ECLASS_VARIABLE: TS_BINDINGS
+# @PRE_INHERIT
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# Array of bindings language to build. Currently only "python" is supported.
+
+for binding in "${TS_BINDINGS[@]}"; do
```

Comment:
I think it'd be better to uppercase and prefix the variable, to avoid it accidentally overwriting something in ebuild.

Reply:
thank you, fixed